api/data_explorer/controllers/export_url_controller.py

```diff
@@ -94,6 +96,14 @@ def _get_entities_dict():
                 'table_name': table_name
             }
         })
+    if query != "" and cohortName != "":
```

Comment:
[Python style](https://github.com/google/styleguide/blob/gh-pages/pyguide.md#2144-decision) is: `if query and cohort_name`: